src/utils/generator.js

```diff
@@ -52,12 +55,159 @@ import type {
   BabelNodeBlockStatement,
   BabelNodeLVal,
 } from "@babel/types";
-import { nullExpression, memberExpressionHelper } from "./babelhelpers.js";
-import { Utils, concretize } from "../singletons.js";
+import { memberExpressionHelper } from "./babelhelpers.js";
+import { concretize, Utils } from "../singletons.js";
 import type { SerializerOptions } from "../options.js";
+import * as t from "@babel/types";
+
+export type OperationDescriptorType =
+  | "IDENTIFIER"
+  | "SINGLE_ARG"
+  | "REBUILT_OBJECT"
+  | "CONSOLE_LOG"
+  | "FOR_IN"
+  | "DO_WHILE"
+  | "CONCRETE_MODEL"
+  | "BINARY_EXPRESSION"
+  | "LOGICAL_EXPRESSION"
+  | "CONDITIONAL_EXPRESSION"
+  | "UNARY_EXPRESSION"
+  | "ABSTRACT_FROM_TEMPLATE"
+  | "GLOBAL_ASSIGNMENT"
+  | "GLOBAL_DELETE"
+  | "EMIT_PROPERTY_ASSIGNMENT"
+  | "ABSTRACT_PROPERTY"
+  | "JOIN_GENERATORS"
+  | "APPEND_GENERATOR"
+  | "DEFINE_PROPERTY"
+  | "PROPERTY_DELETE"
+  | "THROW"
+  | "CONDITIONAL_THROW"
+  | "COERCE_TO_STRING"
+  | "ABSTRACT_FROM_TEMPLATE"
```

Comment:
This is a duplicate. Maybe sorting alphabetically might make sense to catch those.